code/modules/mob/inventory.dm

```diff
@@ -77,25 +71,12 @@
 	W.plane = initial(W.plane)
 	W.dropped()
 
-/mob/proc/drop_item_v()		//this is dumb.
-	if(stat == CONSCIOUS && isturf(loc))
-		return drop_item()
-	return 0
-
-//Drops the item in our left hand
-/mob/proc/drop_l_hand()
-	return unEquip(l_hand) //All needed checks are in unEquip

```

Comment:
It's to keep things simple! And have a short hand for a common action! And exist to be overridden on children! Please leave it alone! If you want to just force unequip on `l_hand`, you can use a force unequip, but leave these alone! Nothing is stopping you from just using `unEquip(l_hand, TRUE)` in place of this, and leaving this proc alone.